### PR TITLE
CLAN6-67-widget-tabs

### PR DESCRIPTION
**Pull request by Cich0sza**

### CODE REVIEW

File: cogboard-webapp/src/components/TabPanel.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (marcin-witaszek) [inline cogboard-webapp/src/components/TabPanel.js]: Pls import PropTypes props separately for consistency with other recently added component files

File: cogboard-webapp/src/components/WidgetForm/index.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+import { sortByKey } from "../helpers";
+import { COLUMNS_MIN, ROWS_MIN } from '../../constants';
+
+import { Box, FormControlLabel, FormControl, MenuItem, TextField, Switch, Tab } from '@material-ui/core';
+import DropdownField from '../DropdownField';
+import WidgetTypeForm from '../WidgetTypeForm';
+import { StyledNumberField, StyledTabPanel, StyledTabs } from './styled';
+import { StyledFieldset } from '../styled';
+
+const renderWidgetTypesMenu = (widgetTypes) =>
```
Comment (marcin-witaszek) [inline cogboard-webapp/src/components/WidgetForm/index.js]: Personally I would move this function expression inside the component function body or to separate `helpers` file in the same directory

Comment (goeson00) [reply cogboard-webapp/src/components/WidgetForm/index.js]: Move it to component helpers.

Comment (Cich0sza) [reply cogboard-webapp/src/components/WidgetForm/index.js]: done :)

**Pull request merged**